samples/neo-pixels-x8/Program.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
we usually put device samples under `src/devices/<yourdevice>/samples` - see other devices for reference